FastAutoAugment/datasets.py

```diff
@@ -0,0 +1,42 @@
+import torchvision
+from collections import defaultdict
+from PIL import Image
+from theconf import Config as C
+
+class CIFAR10_mod(torchvision.datasets.CIFAR10):
+    def __init__(self, root, train=True, transform=None, target_transform=None,
+                 download=False):
+        super().__init__(root, train, transform, target_transform, download)
+        self.hardness_scores = defaultdict(defaultdict)
+        
+    def __getitem__(self, index):
+        """
+        Args:
+            index (int): Index
+
+        Returns:
+            tuple: (image, target) where target is index of the target class.
+        """
+        img, target = self.data[index], self.targets[index]
+        
+        measure = C.get()['hardness']['aug_measure']
+        if measure in self.hardness_scores and index in self.hardness_scores[measure]:
+            hardness_score = self.hardness_scores[measure][index]
+        else:
+            hardness_score = 1
```

Comment:
the else case would only come at the start of the epoch right and never again?

Reply:
Yes. You are right.

Reply:
Apply assertion for `drop_last=False` when using hardness score